ui/pages/[group]/[round]/[bucket]/index.tsx

```diff
@@ -107,13 +106,20 @@ const BucketIndex = ({ round, currentUser, currentGroup, router }) => {
     variables: { id: router.query.bucket },
   });
   const [editImagesModalOpen, setEditImagesModalOpen] = useState(false);
+  const [tab, setTab] = useState(0);
   const { bucket } = data ?? { bucket: null };
   const showBucketReview =
     currentUser?.currentCollMember?.isApproved &&
     round.bucketReviewIsOpen &&
     round.guidelines.length > 0 &&
     bucket?.published;
 
+  const tabsList = useMemo(() => ["bucket", "comments", "funders"], []);
+  useEffect(() => {
+    const index = tabsList.findIndex(tab => tab === router.query.tab);
+    setTab(index > -1 ? index : 0);
+  }, [router.query.tab, tabsList]);
```

Comment:
When I've navigated through the tabs and then change the url by navigating through my history (`ctrl+left/right`) the ui doesn't update. I would think that having the `router.query.tab` here would be enough for that. I'll see if there's an easy fix.

Reply:
I went into a bit of a rabbit hole trying to improve this but the libraries we use were really not cooperating. Next.js isn't very well optimized for query param navigation I think, maybe having it as normal `/[param]` would work better, I think it did in Realities. But this is probably good enough for now so I'll merge.